docs/guides/LocalLLMs.md

```diff
@@ -54,6 +55,28 @@ docker run \
 
 You should now be able to connect to `http://localhost:3000/`
 
+### 2.2 Build from Source
+Use the instructions in [Development.md](/Development.md) to build OpenDevin.
+Make sure `config.toml` is there by running `make setup-config` which will create one for you. In `config.toml`, enter the followings:
+
+```
+LLM_MODEL="ollama/llama3:8b"
+LLM_API_KEY="ollama"
+LLM_EMBEDDING_MODEL="local"
+LLM_BASE_URL="http://localhost:4000"
+WORKSPACE_BASE="./workspace"
+WORKSPACE_DIR="$(pwd)/workspace"
+```
+Replace `LLM_MODEL` of your choice.
+
+To avoid buggy `OpenAI Exception: 404` error, we use `litellm` to route the traffic from `ollama`:
+```
+pip3 install litellm[proxy]
+litellm --model ollama/llama3:8b --api_base http://localhost:11434
```

Comment:
Can you explain more why the litellm proxy is necessary?

Reply:
Mentioned in #1052 ([here](https://github.com/OpenDevin/OpenDevin/issues/1052#issuecomment-2052105359)).
It seems to be a walkaround to this issue:
<img width="434" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/73148791/c9608724-805f-4139-a8e7-c1843482f19a">

Reply:
I've tested in WSL2 as well, using litellm solves the exception perfectly

Reply:
Hmm. I think the issue in #1052 is that the model name isn't being set correctly in OpenDevin. That could be a user error, or it could be a bug inside OpenDevin.

We definitely _shouldn't_ need the litellm proxy, so I'm hesitant to put this into our documentation. Sounds like it's a valuable workaround for now though

Reply:
it seems a walkaround yes. maybe i'll simply this PR to remove the proxy mention

Reply:
tks, and addressed in [5109a72](https://github.com/OpenDevin/OpenDevin/pull/1269/commits/5109a724750cce1b00a60336fd20c3481be17bf9).
I think for walkaround of the issue, it should probably be a bug inside opendevin. I will look into that to see if i can  contribute too~

Reply:
btw, do you think we shd add this walkaround somewhere in opendevin? for example, troubleshoot

Reply:
Looks good! This is hopefully good enough for now

Reply:
Nice one